### PR TITLE
Messages layout redesigned (jQuery UI)

### LINKED ISSUE #846: Message reminders
Go to Messages. Put "Hide reminders", "View log" and "Send dated reminders" button be on the same line. 
- Go to Messages-> Show reminders-> send a dated reminder. Place the header “Messages you have sent today” be under the “Send this message” button.

### PR DESCRIPTION
**Pull request by MigDinny**
Fixes #846

Messages was redesigned (compare the screenshots, I've removed unnecessary stuff, the layout was a mess) and I added jQuery UI's accordion instead of hide and show button system.

Dependencies were added on header file.

dated_reminders.php was correctly idented.

Fixed minor bugs (such as lack of ; ) and minor design/layout issues.

**Before:**

image
_(Show reminders button has some bugs when we click it)_

image
_(unaligned elements)_

**After:**

image
_(Used jQuery UI's accordion and repositioned the elements)_

image
_(Aligned all elements to the center of the page and did this to all styles)_

### CODE REVIEW

File: library/headers.inc.php [hunk 1]
```diff
@@ -70,6 +70,11 @@ function call_required_libraries($library_array){
         <script type="text/javascript" src="<?php echo $GLOBALS['standard_js_path']; ?>jquery-datetimepicker/jquery.datetimepicker.full.min.js"></script>
     <?php
     }
+    if (in_array("jquery-ui",$library_array)){   ?>
+        <link rel="stylesheet" href="//code.jquery.com/ui/1.12.1/themes/base/jquery-ui.css" media="screen" />
```
Comment (teryhill) [inline library/headers.inc.php]: What happens if there is no internet connection? Try to work with the assets in the system, there are jquery files on the system

Comment (MigDinny) [reply library/headers.inc.php]: Even if I put there a system file, internet connection is still needed anyway. Whats the point?

**Comment by aethelwulffe** [comment]
@MigDinny I will test this as soon as I can.  I am hoping it looks good with multiple tabs open...at least when running bits of your theme anyway.

**Comment by MigDinny** [comment]
@aethelwulffe i think so

File: library/headers.inc.php [hunk 2]
```diff
@@ -70,6 +70,11 @@ function call_required_libraries($library_array){
         <script type="text/javascript" src="<?php echo $GLOBALS['standard_js_path']; ?>jquery-datetimepicker/jquery.datetimepicker.full.min.js"></script>
     <?php
     }
+    if (in_array("jquery-ui",$library_array)){   ?>
+        <link rel="stylesheet" href="<?php echo $GLOBALS['css_path']; ?>jquery-ui-1-12-1/jquery-ui.css" media="screen" />
+        <script type="text/javascript" src="<?php echo $GLOBALS['standard_js_path']; ?>jquery-ui-1-12-1/jquery-ui.js"></script>
+    <?php
+    }
```
Comment (aethelwulffe) [inline library/headers.inc.php]: Thanks for adding this.
Maybe we can pop out some more tasks/issues to update the implementation of jquery-ui elsewhere now.

**Comment by aethelwulffe** [comment]
Looks like the CSS file has been added, and as soon as @teryhill approves the changes, any one of us are good to merge this.

File: library/headers.inc.php (hunk 2 above)
Comment (MigDinny) [reply library/headers.inc.php]: This library is really awesome!

**Comment by MigDinny** [comment]
@aethelwulffe 
image
image

is this what you need?

**Comment by MigDinny** [comment]
@aethelwulffe thanks!

**Comment by teryhill** [comment]
@aethelwulffe Says it is ready Merging it

**Comment by teryhill** [comment]
PR merged closing issue

**Pull request merged**